app/models/saved_claim/veteran_readiness_employment_claim.rb

```diff
@@ -217,16 +217,12 @@ def send_central_mail_confirmation_email(user)
     )
   end
 
-  def process_attachments!(user)
+  def process_attachments!(_user)
```

Comment:
If you aren't using the `user` argument you can remove it and also remove it on line 162